game/hud/src/widgets/Crafting/components/Tools/index.tsx

```diff
@@ -88,7 +89,7 @@ class Tools extends React.Component<ToolsProps, ToolsState> {
     const makeInput = (args: MakeInput) => {
       return (
         <Input size={args.size} numeric={args.numeric} min={args.min} max={args.max}
-          onChange={args.change} value={args.get()}/>
+          onChange={args.change} value={args.get()||args.defaultValue}/>
```

Comment:
Nit, not sure why this didn't show up in the tslint filter. Should have white spacing around the ||. Seems like my VSCode tells me that this is an error but the build doesn't. Not sure why...